internal/move2kubeapi/move2kubeapi.go

```diff
@@ -17,430 +17,199 @@ limitations under the License.
-		return
-	}
-	defer file.Close()
-	if err := m2kapp.UploadAsset(name, handler.Filename, file, isCustomization); err != nil {
-		log.Errorf("Could not update with asset : %s", err)
-		w.WriteHeader(http.StatusGone)
-		return
-	}
-	w.WriteHeader(http.StatusOK)
-	_, _ = io.WriteString(w, "File "+handler.Filename+" Uploaded successfully")
-	log.Infof("Asset %s uploaded successfully for app %s", handler.Filename, name)
-}
-
-func getAssetsList(w http.ResponseWriter, r *http.Request, isCustomization bool) {
-	name := mux.Vars(r)[nameRouteVar]
-	assets, err := m2kapp.GetAssetsList(name, isCustomization)
-	if err != nil || assets == nil {
-		log.Errorf("Could not get the assets/customizations for the app %s . Error: %q", name, err)
-		w.WriteHeader(http.StatusNotFound)
-		return
-	}
-	w.WriteHeader(http.StatusOK)
-	w.Header().Set("Content-Type", "application/json")
-	if err := json.NewEncoder(w).Encode(assets); err != nil {
-		log.Errorf("failed to marhsal the assets/customizations list %+v to json. Error: %q", assets, err)
-	}
-}
-
-func getAsset(w http.ResponseWriter, r *http.Request, isCustomization bool) {
-	name := mux.Vars(r)[nameRouteVar]
-	asset := ""
-	if isCustomization {
-		asset = mux.Vars(r)[customizationRouteVar]
-	} else {
-		asset = mux.Vars(r)[assetRouteVar]
-	}
-	file, filename, err := m2kapp.GetAsset(name, asset, isCustomization)
-	if err != nil || file == nil {
-		log.Errorf("Could not get asset %s for the app %s . Error: %q", asset, name, err)
-		w.WriteHeader(http.StatusNotFound)
-		return
-	}
-	w.WriteHeader(http.StatusOK)
-	w.Header().Set("Content-Type", "application/octet-stream")
-	w.Header().Set("Content-Disposition", "attachment; filename="+filename)
-	if _, err := io.Copy(w, file); err != nil {
-		log.Errorf("failed to send the asset/customization to the client. Error: %q", err)
-	}
-}
-
-func deleteAsset(w http.ResponseWriter, r *http.Request, isCustomization bool) {
-	name := mux.Vars(r)[nameRouteVar]
-	asset := ""
-	if isCustomization {
-		asset = mux.Vars(r)[customizationRouteVar]
-	} else {
-		asset = mux.Vars(r)[assetRouteVar]
-	}
-	if err := m2kapp.DeleteAsset(name, asset, isCustomization); err != nil {
-		log.Errorf("Could not delete asset : %s", err)
-		w.WriteHeader(http.StatusNotFound)
-		return
-	}
-	w.WriteHeader(http.StatusOK)
-	_, _ = io.WriteString(w, "Asset "+asset+" deleted successfully")
-	log.Infof("Asset %s deleted successfully for app %s", asset, name)
-}
-
-func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
-	name := mux.Vars(r)[nameRouteVar]
-	plan := r.FormValue("plan")
 
-	artifactName := name + "-" + cast.ToString(time.Now().Unix())
-	log.Infof("Artifact Name:%s", artifactName)
+	// static routes
+	// swagger UI
+	swaggerDir, _ := fs.Sub(assets.SwaggerUI, "swagger")
+	router.Handle("/swagger", http.RedirectHandler("/swagger/", http.StatusMovedPermanently))
+	router.PathPrefix("/swagger/").Handler(http.StripPrefix("/swagger", http.FileServer(http.FS(swaggerDir)))).Methods("GET")
```

Comment:
We can consider serving this as part of the UI

Reply:
done